qiskit/circuit/qpy_serialization.py

```diff
@@ -130,29 +130,48 @@
 which just adds the ``in_circuit`` field which represents whether the register is
 part of the circuit or not.
 
-.. _range_pack:
+.. _qpy_range_pack:
 
 RANGE
 -----
 
 A RANGE is a representation of a ``range`` object. It is defined as:
 
-.. code-block::
+.. code-block:: c
 
     struct {
         int64_t start;
         int64_t stop;
         int64_t step;
     }
 
-.. _version_3:
+.. _qpy_tuple:
+
+TUPLE
+-----
+
+A TUPLE is a reprentation of a ``tuple`` object. As tuples are just fixed length
+containers of arbitrary python objects
+
+A tuple instruction parameter starts with a header defined as:
```

Comment:
Something's gone a bit wrong here.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/7584/commits/b19c1a5a2c1ac9d1ae4cc049911da143b994ec41